schema.json

```diff
@@ -28,7 +86,120 @@
       "type": "object",
       "allOf": [
         {
-          "$ref": "https://schemas.s1seven.com/schema-definitions/v0.0.7/company/company.json#/definitions/Company"
+          "title": "Company",
+          "type": "object",
+          "allOf": [
+            {
+              "type": "object",
+              "properties": {
+                "ZipCode": {
+                  "type": "string"
+                },
+                "City": {
+                  "type": "string"
+                },
+                "Country": {
+                  "description": "The two-letter ISO country code according https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2.",
+                  "type": "string",
+                  "minLength": 2,
+                  "maxLength": 2,
+                  "pattern": "^[A-Z]{2}$",
+                  "examples": ["AT", "DE", "FR", "ES", "PL", "CN"]
+                },
+                "Email": {
+                  "type": "string",
+                  "format": "email"
+                },
+                "AdditionalInformation": {
+                  "description": "An array of additional free text information on the company.",
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "uniqueItems": true,
+                  "minItems": 1
+                }
+              },
+              "oneOf": [
+                {
+                  "properties": {
+                    "Name": {
+                      "type": "string"
+                    }
+                  },
+                  "required": ["Name"]
+                },
+                {
+                  "properties": {
+                    "CompanyName": {
+                      "type": "string"
+                    }
+                  },
+                  "required": ["CompanyName"]
+                }
+              ],
+              "required": ["ZipCode", "City", "Country"],
+              "additionalProperties": true
+            },
+            {
```

Comment:
Here Company is made up of 3 objects, the base Company, CompanyAddress and CompanyIdentifiers. Instead of directly copying them over like you did here, maybe you could copy CompanyAddress and CompanyIdentifiers into the definitions and use a local reference like we did [here](https://github.com/material-identity/EN10168-schemas/blob/eb3e77f0256fbecf26911e1d4dd1e3714c2bfb72/schema.json#L176) in EN10168?

IMO it will make it easier to maintain in the future, as if for example, we want to make a change to CompanyAddress across all the schemas, it will be easier if it is it's own object in the schema definitions.